pkg/catalog/catalog.go

```diff
@@ -49,30 +39,3 @@ func NewMeshCatalog(kubeController k8s.Controller, kubeClient kubernetes.Interfa
 func (mc *MeshCatalog) GetSMISpec() smi.MeshSpec {
 	return mc.meshSpec
 }
-
-func (mc *MeshCatalog) getAnnouncementChannels() []announcementChannel {
-	ticking := make(chan announcements.Announcement)
-	announcementChannels := []announcementChannel{
-		{"MeshSpec", mc.meshSpec.GetAnnouncementsChannel()},
-		{"CertManager", mc.certManager.GetAnnouncementsChannel()},
-		{"IngressMonitor", mc.ingressMonitor.GetAnnouncementsChannel()},
-		{"Ticker", ticking},
-		{"Services", mc.kubeController.GetAnnouncementsChannel(k8s.Services)},
-	}
-
-	if updateAtLeastEvery > 0 {
-		go func() {
-			ticker := time.NewTicker(updateAtLeastEvery)
```

Comment:
@draychev  I have an open item on me to reintroduce it when appropriate.

Reply:
Do we want to track this in an issue?

Reply:
https://github.com/openservicemesh/osm/issues/2103